packages/@lightningjs/ui-components/src/docs/ThemingTutorial.stories.mdx

```diff
@@ -0,0 +1,267 @@
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  SPDX-License-Identifier: Apache-2.0
+-->
+
+import { Meta, Description } from '@storybook/addon-docs';
+
+<Meta title="Docs / Theming/ Tutorial" />
+
+# Building an Application with Theming
+
+Before proceeding, read through the [Theming Overview](../?path=/story/docs-theming-overview--page) documentation to understand the basics of a global theme.
+
+You can see the final result of this example tutorial via this [Lightning Playground link](https://lightningjs.io/playground/#N4IgJgpgDhB2mwMYEsIGcQC5QAEA2yA5gBYAusyshAVmgPSID2AThFiGaVGpnQ2LAB0aANYBPKAENEIwZABudfETIUqtBiwg4ATIICMegAwgANCGUlylGvQCuyALRMAtlEaw4pDJg6kuPHyIAsLiUjJyEIqWqjYaDs6Mbh5eGAC+5miIzMhQ3ligeJJ2SMSCtAUgAGbIeGy+RSWIZRXmTJDsADqwAAR9-X3IycykPQCCUFA9VcxJPQDkgnQTUOVo8wDc3QMD2zt9AO6UYIwHgpL+0sQAkrBoMIikLAAUR-CngnhUAJRbsHv7Jh3UaMPLIDxoHoAXh6wDQpEkhAgmFhBxR+gAnDojKYesR0UYABw4nqIOqSZgAYUYeBYKKMAA8jMyWcy0mk-vs+kD4T1JJNoT1PAdxpNnqDSOC7r8ATsTog7C4vIIAEaMMBic6TOBgSnEWpgZ781bwxEQQRI0iUySweSSNDPb4y3r9MzgaA6uAodBYADaICWKzWIAAuhkQEGKthqrV6hHJsG2uq40N3CMegAZFTWKjTWYuBYxHO2TSsTbdVMsUYAUQZkjcdQACma83NFnRa-WoE2zWty-9PAy06NIFVinhRmT7ZCVj0IAzSDrIVmrGpCIIVgREBcpbDZabJYgegB9RcNi4QR17l37VikOzMXrAWVcnrNpEo4A9UgSZE9TsNhA74QD0aQvv0YE3qB3SyseloAGKMAqaAQIa3zXq+d4Pr0pD6mgggIoQzzzMB8zOgMkFpG6kAwPAXqoD4-pFmuGhMKwbpLAB3ZAb2FRhuYXE9kiwbRjUdTsIJPHCa0IDtCmwyjF+ABC9oQLiSl2P4Hi4gAKvOpBKYwDKga2BbzMxcT2E4rjuJ4sDeP2lbpgAVHykLwmIdSQjMbacXWgHAcIP5eX2fzdPOw49KO46TkUaCQpJwFzguS49CpKEYf0B7IEelonseNkpPZABy9aXuhz5QQMWGPgsiVmv2FEwVB2W5RAozHqexAQEqADKwXlZlt7tdhPQeV5nIQc1AytSeZ7dheV6VZhI21ctr79NS9leJ+4EbT0aIHdCAB8B2mHtG34niJ14udVX7dMdQMrt90Pf0YDIKwjxSii8xMHgiqwPMd1va+1B2PCyBVGIW2LvZv2IF4EDMMDF2vhkaNcjpyCkHUL2g1yP4wCiekLoZDKY-sGOvRtAAi6DZLkkoePjBM7ETf6kwZRmUzs1Ns2lmlPLArMC30HMohpWkDgLkFvXLXIK30kH7gih49HlhEOhVF01b0vq81+sBlb92O4xAwM9FIuG-bDypm3U8ygSDD1GybCz01kORgh4lvW1d8x2-Zgie4zPtA87htCu78xS8LfsXAHQekIIce+9BNMhpNyvTf0x52FAYCLTrNNoAXyOCPnhfF9n7N4YIxkwrhyD4eN5ouBShCUAAGrXAzN-hVdF4uDvlX3-QD5XBfDxAofe8zsCOuP4v10PF5p4v5FTTLefTxeo9LRdk-HqPgjW80zzrftQJw6Qv0AOoQHgrggU8PSMA+b5mgAhKjNMDIdSeIpHDfnrm3QQHdmBd1gN3HorkdAu32jUBkqEUSkGYHYNSvMqhPWuGeT8PRIHQMMlpFwaCwEDQIjjOoABZTulASFPALPzB6bcCGLgXP1Tyf55gfXuEUMQOgnZKz5lvHOO8+hr0XHPJmUpD402PjI8OZ8LgXyvhtG+O0FgAAlkZqQOiBNAxBThjSSCBecXY6jfn0nyeAfIegqiFtpPkeA8A9CKMgMA79NI9E0rUZAAAvGw35GAiDgN5fM3jmCgJ6iBMSEBf6II2oA+uwDQEtyCtwiB9CYFwJ6Ag7ByAUFgDQRgrB-9+g4PnHgnqBCiEMMYKQ8hGTwGQC9rIjwdCoENNIZHCpfQ2GwmsZwgav01Qan0MIi6aQxEZ1gnvRcG95GvmPhvFRpA1G80lObX6AB5AA0vMaZsyVZQTgu1RCyFULLK5HrdJg8N610otRD0dEkAMT9BYAgq5LJ0ASIVOy+RzB+UsVJc0bd8J8XDPVYSEKRKgHiRJfy3FApwpknJdgEUqykghKMFUqlBS4ViTdS+exurZnIbEz4kgxAfxTl7CAcBtEuyOGAG2MSlTUtpZpYQ2RGWwHvi7epMDKWcoEXS7J3SYEu22bQnJjCkiivBeECueBCAuzaWHBeXTiGNKYUq4QKrmANwZAyPA3QZkbBebRBA3pGJhnDAiFUPgQCSFcSANIQA).
```

Comment:
The playground seems to be missing the correct color for the text of the button.
<img width="828" alt="Screen Shot 2023-11-06 at 11 58 55 AM" src="https://github.com/rdkcentral/Lightning-UI-Components/assets/2678766/eae8c829-2860-4ca8-85e7-671da6d04067">

Additionally, you could use "https://lightningjs.io/playground/#KClAjwInYW8n" for the playground, if you want something shorter.

Reply:
Yeahhhh the text color on the Button is a different issue 😬 I'm hoping it's fixed after #368 goes in 🤞

Reply:
Good call on the links, I will shorten all of them, thanks!